pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/reports/stagereports/UploadMetadataVerifyStage.kt

```diff
@@ -16,6 +16,6 @@ class UploadMetadataVerifyStage: SchemaDefinition(priority = -1) {
     var issues: List<String>? = null
 
     companion object {
-        val schemaDefinition = SchemaDefinition(schemaName = "dex-metadata-verify", schemaVersion = "0.0.1")
+        val schemaDefinition = SchemaDefinition(schemaName = "upload-status", schemaVersion = "1.0.0")
```

Comment:
I don't understand why this was changed from metadata verify to upload status.  Would you please explain?

Reply:
This change is reverted for the Metadata classes. 
However, upon closer inspection, noticed that the default SchemaDefinition values should be set to the following:
_companion object {
        val schemaDefinition = SchemaDefinition(schemaName = "upload-status", schemaVersion = "1.0.0")
    }_

The content_schema_name and content_schema_version for the upload is updated to "upload-status", "1.0.0" respectively in the reports.py scripts